src/main/java/org/opensearch/knn/plugin/stats/KNNGraphValue.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.plugin.stats;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Contains a map of counters to keep track of different values
```

Comment:
Can you pls elaborate the JavaDoc and mention that this eum Class is related to the KNNGraph related metrics